spec.bs

```diff
@@ -345,9 +345,11 @@ This is detectable because it can change the set of fields that are read from th
 1. [=parallel queue/enqueue steps|Enqueue the following steps=] to |queue|:
   1. Let |permission| be the result of [=checking interest group permissions=] with
     |interestGroup|'s [=interest group/owner=], |frameOrigin|, and "`join`".
-  1. If |permission| is false, then [=queue a task=] to [=reject=] |p| with a
-     "{{NotAllowedError}}" {{DOMException}} and abort these steps.
-  1. [=Queue a task=] to [=resolve=] |p| with `undefined`.
+  1. If |permission| is false, then [=queue a global task=] on [=DOM manipulation task source=],
+    given |global|, to [=reject=] |p| with a "{{NotAllowedError}}" {{DOMException}} and do not run

```

Comment:
"abort these steps" should still be kept. I'm not sure why it should be deleted.

Reply:
I saw "do not run the remaining steps" in some specs (or maybe some PR comments) and thought it was a little bit more clear, so changed "abort these steps" to it. I changed it back to "abort these steps", but let me know if "do not run the remaining steps" is preferred.

Reply:
"abort these steps" is generally preferred. I guess https://github.com/whatwg/infra/pull/255 and https://github.com/whatwg/infra/issues/258 have some examples, but generally this is the verbiage you find in many specs.